docs/writing-stories/introduction.md

```diff
@@ -138,6 +146,8 @@ Refine this pattern by introducing `args` for your component's stories. It reduc
     'react/button-story-using-args.ts.mdx',
     'vue/button-story-using-args.2.js.mdx',
     'vue/button-story-using-args.3.js.mdx',
+    'vue/button-story-using-args.2.ts.mdx',
+    'vue/button-story-using-args.3.ts.mdx',
```

Comment:
@valentinpalkovic if you could rename the files and update the references in the documentation to for instance: "vue/button-story-using-args.ts-2.ts.mdx" for Vue 2 and "vue/button-story-using-args.ts-3.ts.mdx" for Vue 3,   I'd appreciate it. The reason is that if you spin up the site with the file naming you've used it will yield the following issue with the documentation:

![files-render-issue](https://user-images.githubusercontent.com/22988955/164808770-d7290974-a30f-48b3-aedc-a42eb5b95eb7.png)

Renaming them, it will render them correctly and select the correct snippet, as you can see below:

![files-render-ok](https://user-images.githubusercontent.com/22988955/164810891-9fe4b6c6-64a2-4f2d-8058-2f2b4dd2a7ab.png)

Let me know once you've updated it and we'll go from there.

Reply:
@jonniebigodes Updated!

Reply:
That's great, thanks for making the adjustments!